src/simulation/m_mpi_proxy.fpp

```diff
@@ -2152,19 +2226,24 @@ contains
         ! Send/Recv
         #:for rdma_mpi in [False, True]
             if (rdma_mpi .eqv. ${'.true.' if rdma_mpi else '.false.'}$) then
+                p_send => c_divs_buff_send(0)
+                p_recv => c_divs_buff_recv(0)
+
                 #:if rdma_mpi
```

Comment:
is `rdma_mpi` checked to be working in any of the tests for mi250x or nvidia GPUs? seems like it should be

Reply:
If I ran the 2-rank test cases on Frontier and Wingtip with rdma_mpi ON, would that be sufficient for you?

Reply:
This would indeed be nice but I don't think this PR should be the one to fix it.